self-hosted/backup-and-restore/logical-backup.md

```diff
@@ -0,0 +1,188 @@
+
+   This includes `psql`, and `pg_dump`.
+
+## Back up and restore an entire database
+
+You backup and restore an entire database using `pg_dump` and `psql`. in Terminal: 
+
+<Procedure>
+
+In Terminal:
+
+1. **Set your connection strings**
+
+   These variables hold the connection information for the source database to backup from and
+   the target database to restore to:
+
+   ```bash
+   export SOURCE=postgres://<user>:<password>@<source host>:<source port>/<db_name>
+   export TARGET=postgres://<user>:<password>@<source host>:<source port>
+   ```
+
+1. **Backup your database**
+
+   ```bash
+   pg_dump -d "$SOURCE" \
+     -Fc -f <db_name>.bak  
+   ```
+    You may see some errors while `pg_dump` is running. See [Troubleshooting self-hosted TimescaleDB][troubleshooting]
+    to check if they can be safely ignored.
+
+1. **Restore your database from the backup**
+
+   1. Connect to your target database:
+      ```bash
+      psql -d "$TARGET"
+      ```
+
+   1. Create a new database and enable TimescaleDB:
+
+      ```sql
+      CREATE DATABASE <restoration database>;
+      \c <restoration database>
+      CREATE EXTENSION IF NOT EXISTS timescaledb;
+      ```
+   
+   1. Put your database in the right state for restoring:
+
+       ```sql
+       SELECT timescaledb_pre_restore();
+       ```
+
+   1. Restore the database:
+
+      ```sql
+       \! pg_restore -Fc -d <restoration database> <db_name>.bak
+       ```
+      
+   1. Return your database to normal operations: 
+
+      ```sql
+      SELECT timescaledb_post_restore();
+      ```
+      Do not use `pg_restore` with the `-j` option. This option does not correctly restore the 
+      TimescaleDB catalogs.
+
+</Procedure>
+
+
+## Back up and restore individual hypertables
+
+`pg_dump` provides flags that allow you to specify tables or schemas
+to back up. However, using these flags means that the dump lacks necessary
+information that TimescaleDB requires to understand the relationship between
+them. Even if you explicitly specify both the hypertable and all of its
+constituent chunks, the dump would still not contain all the information it
+needs to recreate the hypertable on restore.
+
+To backup individual hypertables, backup the database schema, then backup only the tables 
+you need. You also use this method to backup individual plain tables.
+
+<procedure>
+In Terminal:
+
+1. **Set your connection strings**
+
+   These variables hold the connection information for the source database to backup from and
+   the target database to restore to:
+
+   ```bash
+   export SOURCE=postgres://<user>:<password>@<source host>:<source port>/<db_name>
+   export TARGET=postgres://<user>:<password>@<source host>:<source port>/<db_name>
+   ```
+
+1. **Backup the database schema and individual tables**
+
+   1. Back up the hypertable schema:
+
+      ```bash
+      pg_dump -s -d $SOURCE --table conditions -N _timescaledb_internal | \
```

Comment:
I couldn't' t understand the reason for adding `-N _timescaledb_internal | grep -v _timescaledb_internal`. Since we recommend creating hypertable on target again, we don't really need to dump the schema of chunks from `_timescaledb_internal`.

Reply:
Removed.